sample/playableGame.py

```diff
@@ -0,0 +1,52 @@
+import sample.board as board
+board = board.Board()
+
+def get_next_sub_board():
+    if board.next_move == (-1,-1):
+        #get user entered subX,subY
```

Comment:
I would put this all in a function called ask_player_for_sub_board() or something